### DISCUSSION TITLE
Create an education sub domain

### DISCUSSION THREAD

**Discussion by danirabbit** [elementary/website#2767] [Ideas]
## Problem

In the education space, it seems Google/Chromebook and Apple/iPad are pushed pretty heavily as solutions bundles. We should have some kind of proposed solution for educators to adopt elementary OS in the classroom instead 

## Proposal

At a bare minimum we should have an `edu` and/or `education` subdomain that describes how elementary OS can be used for education. Perhaps this includes links to low cost hardware vendors like Pine and Raspberry Pi and privacy oriented service providers like FastMail, and instructions for installing the latest version of office suites etc. 

## Prior Art

We should look into where elementary OS has been previously deployed in education such as in Ville de Fontaine and find out what a complete education solution looks like. 

We could also look at https://www.apple.com/education/k12/ and https://edu.google.com/ to see how our major competitors position themselves

**Comment by btkostner** [comment]
This could also be a nice spot to show off some apps that could be used in education as well

**Comment by danirabbit** [comment]
Yeah if we could pull a carousel of like top apps from education, math, science categories or something that would be dope

**Comment by cassidyjames** [comment]
I wonder if this would be more generalizable as a sub-page rather than a whole subdomain—e.g. we might want to have pages dedicated to other specific use cases/domains in the future.

**Comment by swirly** [comment]
If you intend to push an educational use, you should get in touch with 2 projects : [sambaedu](https://www.sambaedu.org/) and [eole](https://pcll.ac-dijon.fr/eole/). Those two projects have domain controllers based on samba4 ad with a procedure for integrating linux client. You certainly could collaborate for the integration of elementary as a client in those systems.
They are the two main systems for deploying linux in french "college" and "lycee"

**Comment by MrPotatoBobx** [comment]
I am no expert but tackling with giants such as google and apple will be for sure hard, I think we need to first establish what sets us apart from competitors and our weaknesses

I think we should first tackle issues such as:
- https://github.com/elementary/triage/issues/46
 - In the free and opensource world, you have many choices, instead of being locked to a product made by the provider of the OS and or hardware, I think we should make some kind of comparison or guide to help them choose some of the tools they will need
 - We possibly need to include support like RHEL does, both google and apple provide some kind of enterprise package that has support and an easy way to manage an array of devices
 - It should be low maintenance, a good way to do this is sell cheap devices that ship with ElementaryOS, then the operating system will be optimized for the hardware, and maybe some more testing, and an administrator dashboard to change system settings in the mass and monitor hardware, other than those ideas, I already think that ElementaryOS is fast and stable 

Those are just some of my ideas, would love to see your opinions!

**Comment by janxkoci** [comment]
This is cool! :grin: 

Just FYI one nice office package is OnlyOffice. It's an open source project with a freemium business model (local apps and self-hosted cloud is free). It provides local & cloud apps, integrations with ownCloud & Nextcloud and so on. It's been even used in [education institutions](https://www.onlyoffice.com/en/customers.aspx)!

The integration to elementary OS is not the best at the moment (https://github.com/ONLYOFFICE/DesktopEditors/issues/216) but maybe it will improve in the future. And it's definitely more user friendly than LibreOffice!
